### DISCUSSION TITLE
MenuBar: support for autoHideDelay, autoHide and autoDisplay like PrimeNG

### DISCUSSION THREAD

**Discussion by srikanth-nallamothu** [primefaces/community#649] [PrimeReact] [answered]
When I mouseover the menu item sub menu is being opened, but when I move diagonally to click on sub menu item, it is highlighting the next menu subitems. Which can be avoided if we have support to `autoHideDelay` , `autoHide` and `autoDisplay`. These props are currently being supported in primeng. Requesting to have support of the same in prime react. If these are already supported please point me with links.

Prime ng: https://primeng.org/menubar#api.menubar.props.autoHideDelay
Primereact: https://primereact.org/menubar/#api.Menubar.props

**Comment by melloware** [comment, marked as answer]
Issue created: https://github.com/primefaces/primereact/issues/5563
